.gitignore

```diff
@@ -8,3 +8,4 @@ dist/
 __pycache__/
 *.so
 *~
+.pytest_cache/
```

Comment:
This will change for every user and is generated every time you run the tests so you generally don't check it in